compliance/unapproved_instance_types/README.md

```diff
@@ -0,0 +1,38 @@
+## Unapproved instance types
+ 
+### What it does
+
+This policy Report on any instances that are running using instance types that are not approved. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/approved_instance_types) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more.
```

Comment:
the link to the readme doesn't match the directory where the files are stored.

Reply:
I have made the required changes.